Test failure integration_tests/job_queue/test_lsf_driver.py::test_run_mocked_lsf_queue[success-using_scheduler]

Occured in: 
https://github.com/equinor/ert/actions/runs/8566664397/job/23476858781?pr=7568#step:6:77

```
=================================== FAILURES ===================================
______________ test_run_mocked_lsf_queue[success-using_scheduler] ______________
[gw1] linux -- Python 3.12.2 /opt/hostedtoolcache/Python/3.12.2/x64/bin/python

request = <FixtureRequest for <Function test_run_mocked_lsf_queue[success-using_scheduler]>>
using_scheduler = True

    @pytest.mark.usefixtures(
        "copy_lsf_poly_case",
        "mock_bsub",
        "mock_bjobs",
        "mock_start_server",
    )
    @pytest.mark.integration_test
    def test_run_mocked_lsf_queue(request, using_scheduler):
        if "fail" in request.node.name and using_scheduler:
            pytest.skip(
                "Python LSF driver does not support general resubmission on bsub errors"
            )
>       run_cli(
            ENSEMBLE_EXPERIMENT_MODE,
            "poly.ert",
        )

/home/runner/work/ert/ert/tests/integration_tests/job_queue/test_lsf_driver.py:184: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/home/runner/work/ert/ert/tests/integration_tests/run_cli.py:16: in run_cli
    res = cli_runner(parsed)
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/ert/cli/main.py:135: in run_cli
    model.reraise_exception(ErtCliError)
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/ert/run_models/base_run_model.py:320: in reraise_exception
    raise exctype(self.getFailMessage()).with_traceback(
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/ert/run_models/base_run_model.py:257: in start_simulations_thread
    run_context = self.run_experiment(
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/ert/run_models/ensemble_experiment.py:87: in run_experiment
    self._evaluate_and_postprocess(prior_context, evaluator_server_config)
/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/ert/run_models/base_run_model.py:531: in _evaluate_and_postprocess
    self.checkHaveSufficientRealizations(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

num_successful_realizations = 0, minimum_realizations = 1

    @staticmethod
    def checkHaveSufficientRealizations(
        num_successful_realizations: int, minimum_realizations: int
    ) -> None:
        if num_successful_realizations == 0:
>           raise ErtRunError("Experiment failed! All realizations failed!")
E           ert.cli.main.ErtCliError: Experiment failed! All realizations failed!
E           Exception occurred during Task-21
E           Exception in scheduler task: [Errno 2] No such file or directory: PosixPath('bhist')
E           Traceback: Traceback (most recent call last):
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/ert/scheduler/lsf_driver.py", line 297, in poll
E               bhist_states = await self._poll_once_by_bhist(missing_in_bjobs_output)
E                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/site-packages/ert/scheduler/lsf_driver.py", line 357, in _poll_once_by_bhist
E               process = await asyncio.create_subprocess_exec(
E                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/asyncio/subprocess.py", line 224, in create_subprocess_exec
E               transport, protocol = await loop.subprocess_exec(
E                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/asyncio/base_events.py", line 17[42](https://github.com/equinor/ert/actions/runs/8566664397/job/23476858781?pr=7568#step:6:43), in subprocess_exec
E               transport = await self._make_subprocess_transport(
E                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/asyncio/unix_events.py", line 211, in _make_subprocess_transport
E               transp = _UnixSubprocessTransport(self, protocol, args, shell,
E                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/asyncio/base_subprocess.py", line 36, in __init__
E               self._start(args=args, shell=shell, stdin=stdin, stdout=stdout,
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/asyncio/unix_events.py", line 820, in _start
E               self._proc = subprocess.Popen(
E                            ^^^^^^^^^^^^^^^^^
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/subprocess.py", line 1026, in __init__
E               self._execute_child(args, executable, preexec_fn, close_fds,
E             File "/opt/hostedtoolcache/Python/3.12.2/x64/lib/python3.12/subprocess.py", line 19[53](https://github.com/equinor/ert/actions/runs/8566664397/job/23476858781?pr=7568#step:6:54), in _execute_child
E               raise child_exception_type(errno_num, err_msg, err_filename)
E           FileNotFoundError: [Errno 2] No such file or directory: PosixPath('bhist')
E           
E           Exception occurred during Task-22
E           Exception occurred during Task-24
E           Exception occurred during Task-31
E           LSF kill failed due to missing jobid for realization 1
E           LSF kill failed due to missing jobid for realization 3
E           LSF kill failed due to missing jobid for realization 4
E           LSF kill failed due to missing jobid for realization 5
E           LSF kill failed due to missing jobid for realization 6
E           LSF kill failed due to missing jobid for realization 7
E           LSF kill failed due to missing jobid for realization 8
E           unexpected exception in ensemble

/opt/hostedtoolcache/Python/3.12.2/x[64](https://github.com/equinor/ert/actions/runs/8566664397/job/23476858781?pr=7568#step:6:65)/lib/python3.12/site-packages/ert/run_models/base_run_model.py:367: ErtCliError

```

--- a/src/ert/scheduler/lsf_driver.py
+++ b/src/ert/scheduler/lsf_driver.py
@@ -274,9 +274,10 @@ class LsfDriver(Driver):
             if not self._jobs.keys():
                 await asyncio.sleep(self._poll_period)
                 continue
+            current_jobids = list(self._jobs.keys())
             process = await asyncio.create_subprocess_exec(
                 self._bjobs_cmd,
-                *self._jobs.keys(),
+                *current_jobids,
                 stdout=asyncio.subprocess.PIPE,
                 stderr=asyncio.subprocess.PIPE,
             )
@@ -290,7 +291,7 @@ class LsfDriver(Driver):
                 )
             bjobs_states = _Stat(**parse_bjobs(stdout.decode(errors="ignore")))
 
-            if missing_in_bjobs_output := set(self._jobs) - set(
+            if missing_in_bjobs_output := set(current_jobids) - set(
                 bjobs_states.jobs.keys()
             ):
                 logger.debug(f"bhist is used for job ids: {missing_in_bjobs_output}")

